### PR TITLE
CIV-7614 CCD to CUI transform pt2

### PR DESCRIPTION
**Pull request by laperla11**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-7614

### Change description ###
- Transform from CCD back to CUI for 'Part Admit as Paid' and 'Part Admit as Not Paid' response types.
- Remove some lint and build errors

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/models/civilClaimResponse.ts [hunk 1]
```diff
@@ -101,6 +103,10 @@ export interface CCDClaim extends ClaimUpdate {
   specAoSApplicantCorrespondenceAddressRequired?: YesNoUpperCamelCase;
   claimantUserDetails?: IdamUserDetails;
```
Comment (laperla11) [inline src/main/common/models/civilClaimResponse.ts:104]: CIV-7614 changes

File: src/main/common/models/claim.ts [hunk 2]
```diff
@@ -89,14 +91,19 @@ export class Claim {
   respondent1ResponseDate?: Date;
   claimBilingualLanguagePreference: ClaimBilingualLanguagePreference;
   id: string;
```
Comment (laperla11) [inline src/main/common/models/claim.ts:93]: CIV-7614 changes

File: src/main/services/features/response/admission/howMuchHaveYouPaidService.ts [hunk 3]
```diff
@@ -15,6 +15,10 @@ class HowMuchHaveYouPaidService {
       const totalClaimAmount = claim?.totalClaimAmount;
       if (type === ResponseType.PART_ADMISSION) {
```
Comment (laperla11) [inline src/main/services/features/response/admission/howMuchHaveYouPaidService.ts:16]: CIV-7614 changes

File: src/main/services/translation/convertToCUI/convertToCUIEvidence.ts [hunk 4]
```diff
@@ -3,14 +3,14 @@ import {CCDEvidence, CCDEvidenceType} from 'models/ccdResponse/ccdEvidence';
 import {Evidence} from 'form/models/evidence/evidence';
 import {convertToEvidenceType} from 'models/evidence/evidenceType';
```
Comment (laperla11) [inline src/main/services/translation/convertToCUI/convertToCUIEvidence.ts:4]: CIV-7614 changes

File: src/main/services/translation/convertToCUI/convertToCUIPartialAdmission.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
+import {HowMuchHaveYouPaid} from 'common/form/models/admission/howMuchHaveYouPaid';
```
Comment (laperla11) [inline src/main/services/translation/convertToCUI/convertToCUIPartialAdmission.ts:1]: CIV-7614 changes

File: src/main/services/translation/response/convertToCCDRecurringIncome.ts [hunk 6]
```diff
@@ -68,7 +68,7 @@ const toCCDRecurringIncomeItem = (transactionSource: TransactionSource, incomeTy
 const toCCDRecurringIncomeOtherItem = (otherTransactions: TransactionSource[], incomeType: CCDIncomeType): CCDRecurringIncome[] => {
   if (!otherTransactions?.length) return undefined;
```
Comment (laperla11) [inline src/main/services/translation/response/convertToCCDRecurringIncome.ts:69]: CIV-7614 changes

File: src/test/unit/services/features/claimantResponse/claimantSuggestedInstalmentsService.test.ts [hunk 7]
```diff
@@ -62,7 +62,7 @@ describe('Claiman Suggested Instalments Plan Service', () => {
         const repaymentPlan = {
           paymentAmount: PAYMENT_AMOUNT,
```
Comment (laperla11) [inline src/test/unit/services/features/claimantResponse/claimantSuggestedInstalmentsService.test.ts:63]: CIV-7614 changes

File: src/test/unit/services/translation/convertToCUI/convertToCUIPartialAdmission.test.ts [hunk 8]
```diff
@@ -0,0 +1,389 @@
+import {
```
Comment (laperla11) [inline src/test/unit/services/translation/convertToCUI/convertToCUIPartialAdmission.test.ts:1]: CIV-7614 changes

**Review by TatsyMaslova — APPROVED** [review]

**Review by portizdev — CHANGES_REQUESTED** [review]

File: src/main/common/models/ccdResponse/ccdRespondentLiPResponse.ts [hunk 9]
```diff
@@ -1,9 +1,11 @@
...
-  CCDFinancialDetailsLiP,
-} from 'models/ccdResponse/ccdFinancialDetailsLiP';
+import {YesNoUpperCamelCase} from 'form/models/yesNo';
+import {CCDFinancialDetailsLiP} from 'models/ccdResponse/ccdFinancialDetailsLiP';
 import {CCDMediation} from 'models/ccdResponse/ccdMediation';
 
 export interface CCDRespondentLiPResponse {
-  respondent1LiPFinancialDetails?: CCDFinancialDetailsLiP,
-  respondent1MediationLiPResponse?: CCDMediation,
+  partialAdmissionAlreadyPaid?: YesNoUpperCamelCase;
```
[the remark this answers is not in the document]
Comment (laperla11) [reply src/main/common/models/ccdResponse/ccdRespondentLiPResponse.ts:6]: Thanks Pablo, I'm in contact with Kamil about it

[the remark this answers is not in the document]
Comment (laperla11) [reply src/main/common/models/ccdResponse/ccdRespondentLiPResponse.ts]: done

**Review by portizdev — DISMISSED** [review]

**Review by hmcts-version1-domini — DISMISSED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**